<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KreuzstichGUI::MainWindow</class>
 <widget class="QMainWindow" name="KreuzstichGUI::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kreuzstich</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/thimble.png</normaloff>:/icons/thimble.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_Kreuzstich"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_empty_project"/>
    <addaction name="actionNew_project_from_picture"/>
    <addaction name="action_Open_existing_project"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_all"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="actionClo_se_all"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Tile"/>
    <addaction name="action_Cascade"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menu_Generate">
    <property name="title">
     <string>&amp;Generate</string>
    </property>
    <addaction name="actionSave_picture"/>
    <addaction name="actionGenerate_grid"/>
    <addaction name="actionGenerate_script_as_text"/>
    <addaction name="actionGenerate_script_as_LaTeX"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Generate"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_About_Kreuzstich">
   <property name="text">
    <string>&amp;About Kreuzstich</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_New_empty_project">
   <property name="text">
    <string>&amp;New empty project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_project_from_picture">
   <property name="text">
    <string>New project from &amp;picture</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Open_existing_project">
   <property name="text">
    <string>&amp;Open existing project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionClo_se_all">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="action_Cascade">
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="text">
    <string>&amp;Next</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="text">
    <string>&amp;Previous</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionGenerate_grid">
   <property name="text">
    <string>Generate &amp;grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGenerate_script_as_text">
   <property name="text">
    <string>Generate script as &amp;text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionGenerate_script_as_LaTeX">
   <property name="text">
    <string>Generate script as &amp;LaTeX</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave_picture">
   <property name="text">
    <string>Save pict&amp;ure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_About_Kreuzstich</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>openAboutBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClo_se_all</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeAllSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_project_from_picture</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>newProjectFromPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_all</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>saveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>saveProjectAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeActiveSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_New_empty_project</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>newEmptyProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open_existing_project</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>openProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Save</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>saveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>subWindowActivated(QMdiSubWindow*)</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>subwindowActivated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Next</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activateNextSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Previous</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activatePreviousSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cascade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Tile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_in</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_out</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_picture</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>savePicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGenerate_grid</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>generateGrid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGenerate_script_as_text</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>generateScriptAsText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGenerate_script_as_LaTeX</sender>
   <signal>triggered()</signal>
   <receiver>KreuzstichGUI::MainWindow</receiver>
   <slot>generateScriptAsLaTeX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openAboutBox()</slot>
  <slot>newEmptyProject()</slot>
  <slot>newProjectFromPicture()</slot>
  <slot>openProject()</slot>
  <slot>saveProject()</slot>
  <slot>saveProjectAs()</slot>
  <slot>saveAll()</slot>
  <slot>subwindowActivated()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>savePicture()</slot>
  <slot>generateGrid()</slot>
  <slot>generateScriptAsText()</slot>
  <slot>generateScriptAsLaTeX()</slot>
 </slots>
</ui>
